threat_intel/__init__.py

```diff
@@ -22,6 +22,8 @@
     * Domains related to an IP
     * Domain tagging dates for a domain
     * DNS RR history for a domain
+    * WHOIS information for a domain
+    * Latest malicious domains for an IP
 
```

Comment:
Add to the list:
Historical WHOIS information for a domain
WHOIS information for a nameserver
WHOIS information for an email